src/modules/services/submodule/assets/views/ViewServicesAssetsModifySupplyChange.vue

```diff
@@ -156,23 +170,56 @@ export default {
-    const moreThanOneAccount = computed(()=> (appStore.getWalletByName(appStore.state.currentLoggedInWallet.name).accounts.length > 1)?true:false);
+    const supply = ref('0');
+    const accounts = computed( () => walletState.currentLoggedInWallet.accounts);
+    const moreThanOneAccount = computed(()=> (walletState.currentLoggedInWallet.accounts.length > 1)?true:false);
+    const transactionFee = ref('0.000000');
+
+    const ownerPublicAccount = ref(WalletUtils.createPublicAccount(walletState.currentLoggedInWallet.selectDefaultAccount().publicKey, networkState.currentNetworkProfile.network.type));
 
     const changeSelection = (i) => {
       selectedAccName.value = i.name;
       selectedAccAdd.value = i.address;
       balance.value = i.balance;
-      // (balance.value < rentalFee.value)?showNoBalance.value = true:showNoBalance.value = false;
+      (balance.value < rentalFee.value)?showNoBalance.value = true:showNoBalance.value = false;
       showMenu.value = !showMenu.value;
       currentSelectedName.value = i.name;
       isMultiSigBool.value = isMultiSig(i.address);
+      ownerPublicAccount.value = WalletUtils.createPublicAccount(i.publicKey, networkState.currentNetworkProfile.network.type);
     }
 
+    const assetSupply = ref(0);
+    const assetDuration = ref('0 Day');
+    const assetDivisibility = ref(0);
+    const assetTransferable = ref(false);
+    const assetMutable = ref(false);
+
+    const changeAsset = (assetId) => {
+      const selectedAsset = walletState.currentLoggedInWallet.selectDefaultAccount().assets.find((asset) => asset.idHex === assetId);
+      assetSupply.value =Helper.amountFormatterSimple(selectedAsset.amount, selectedAsset.divisibility);
+      if(selectedAsset.duration){
+        assetDuration.value = selectedAsset.duration = ' Day' + ((selectedAsset.duration>1)?'s':'');
+      }else{
+        assetDuration.value = 'No expiration date';
+      }
+      assetDivisibility.value = selectedAsset.divisibility;
+      assetTransferable.value = selectedAsset.transferable;
+      assetMutable.value = selectedAsset.supplyMutable;
+      transactionFee.value = Helper.amountFormatterSimple(AssetsUtils.getMosaicSupplyChangeTransaction(networkState.currentNetworkProfile.network.type, networkState.currentNetworkProfile.generationHash, ownerPublicAccount.value, assetId, selectIncreaseDecrease.value, supply.value), networkState.currentNetworkProfile.network.currency.divisibility);
+    };
+
     const assetOptions = () => {
-      let asset = [];
-      return asset;
+      let assetSelection = [];
+      const filterAsset = walletState.currentLoggedInWallet.selectDefaultAccount().assets.filter((asset) => asset.owner === walletState.currentLoggedInWallet.selectDefaultAccount().publicKey);
+      if(filterAsset.length > 0){
+        filterAsset.forEach((asset) => {
+          assetSelection.push({
+            label: asset.idHex + ' > ' + Helper.amountFormatterSimple(asset.amount, asset.divisibility) + ' XPX',
```

Comment:
not need to convert asset amount, remove `XPX`

Reply:
Okay noted with thanks